src/components/storage/SortSelector.tsx

```diff
@@ -0,0 +1,54 @@
+'use client';
```

Comment:
very unnecessary comment but maybe we should have an empty line everywhere the 'use client' directive is used